apps/fz_http/lib/fz_http_web/auth/json/authentication.ex

```diff
@@ -18,12 +18,10 @@ defmodule FzHttpWeb.Auth.JSON.Authentication do
 
   @impl Guardian
   def resource_from_claims(%{"api" => api_token_id}) do
-    with %ApiTokens.ApiToken{} = api_token <- ApiTokens.get_unexpired_api_token(api_token_id),
+    with {:ok, %ApiTokens.ApiToken{} = api_token} <-
```

Comment:
Do we want HTTP 404 if the token is invalid or HTTP 401? Right now it will be 404. Maybe we should write some tests for this module to enforce the contract there?